atomium/templates/exposed_filters/exposed_filters.component.inc

```diff
@@ -0,0 +1,56 @@
+<?php
+
+/**
+ * @file
+ * Contains component file.
+ */
+
+/**
+ * Implements hook_atomium_theme_hook().
+ */
+function atomium_atomium_theme_exposed_filters(array $existing, $type, $theme, $path) {
+  return array(
+    'exposed_filters' => array(
+      'render element' => 'form',
+      'template' => 'exposed-filters',
+    ),
+  );
+}
+
+/**
+ * Implements hook_preprocess_hook().
+ */
+function atomium_preprocess_exposed_filters(array &$variables, $hook) {
+  $form = $variables['form'];
+  $exposed_filters = array();
+
+  if (isset($form['current'])) {
+    $items = array();
+
+    foreach (element_children($form['current']) as $key) {
+      $items[] = $form['current'][$key];
+    }
+
+    $exposed_filters = array(
+      '#theme' => 'item_list',
+      '#items' => $items,
+      '#attributes' => array(
+        'class' => array(
+          'clearfix',
+          'current-filters',
+        ),
+      ),
+    );
+
+    _atomium_extend_theme_property(
+      $exposed_filters,
+      array(
+        $variables['theme_hook_original'],
+      )
+    );
+  }
+
+  $variables[$variables['base_hook']] = array(
```

Comment:
Why this part is needed?

Reply:
Before rendering the template, Drupal extract the `$variables` and as we uses `$exposed_filters` in the template, we need to do that.

See: https://api.drupal.org/api/drupal/includes%21theme.inc/function/theme_render_template/7.x

Reply:
Cheers!